easybuild/easyconfigs/w/Wayland/Wayland-1.23.92-GCCcore-14.2.0.eb

```diff
@@ -0,0 +1,70 @@
+# Author: Jasper Grimm (UoY)
+# URL of Wayland download changed to GitLab due to changes upstream
+# Author: J. Sassmannshausen (Imperial College London/UK)
+easyblock = 'Bundle'
+
+name = 'Wayland'
+version = '1.23.92'
+
+homepage = 'https://wayland.freedesktop.org/'
+description = """
+Wayland is a project to define a protocol for a compositor to talk to
+ its clients as well as a library implementation of the protocol.  The
+ compositor can be a standalone display server running on Linux kernel
+ modesetting and evdev input devices, an X application, or a wayland
+ client itself.  The clients can be traditional applications, X servers
+ (rootless or fullscreen) or other display servers.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '14.2.0'}
+
+builddependencies = [
+    ('CMake', '3.31.3'),
+    ('Meson', '1.6.1'),
+    ('Ninja', '1.12.1'),
+    ('pkgconf', '2.3.0'),
+    ('binutils', '2.42'),
+]
+
+dependencies = [
+    ('libffi', '3.4.5'),
+    ('expat', '2.6.4'),
+    ('libxml2', '2.13.4'),
+]
+
+default_easyblock = 'MesonNinja'
+default_component_specs = {
+    'start_dir': '%(namelower)s-%(version)s',
+}
+
+components = [
+    ('wayland', version, {
+        'source_urls': ['https://gitlab.freedesktop.org/wayland/%(namelower)s/-/releases/%(version)s/downloads'],
+        'checksums': ['05b3e1574d3e67626b5974f862f36b5b427c7ceeb965cb36a4e6c2d342e45ab2'],
```

Comment:
This checksum is incorrect now.
Also it doesn't match
```
https://gitlab.freedesktop.org/wayland/wayland/-/releases/1.23.92/downloads/wayland-1.23.92.tar.xz.sha256sum
```

Reply:
That checksum is for Wayland 1.23.0.